### PR TITLE
echo -n

### PR DESCRIPTION
**Pull request by evanrichter**
at a shell, echo -n suppresses the newline. I propose that ?en also do this! but this code has a problem, as fflush(stdout); doesn't work. for now, it does what I want, which is add information in-line to other output.

### CODE REVIEW

File: libr/core/cmd_help.c [hunk 1]
```diff
@@ -493,12 +493,24 @@ static int cmd_help(void *data, const char *input) {
...
-		free (newmsg);
+		if (input[1] == 'n') {
+                // mimic echo -n
+                // TODO: doesn't show output unless more output follows (e.g. ?en test;px;)
+                    const char *msg = r_str_chop_ro (input+2);
+                    // TODO: replace all ${flagname} by its value in hexa
+                    char *newmsg = filter_flags (core, msg);
+                    r_str_unescape (newmsg);
+                    r_cons_print (newmsg);
+                    //fflush (stdout); doesn't work
```
Comment (radare) [inline libr/core/cmd_help.c]: Use rconsflush

Comment (radare) [inline libr/core/cmd_help.c]: But this flush is done by the prompt already

Comment (evanrichter) [inline libr/core/cmd_help.c]: I tried that as well

**Comment by radare** [comment]
having echo -n makes no sense in the prompt. because the prompt will eat everything that has no \n, its already printed to stdtout but you cant see it because of this. depending on the needs you have maybe is better to just use eprintf

**Comment by radare** [comment]
can you share an use case for this?

**Comment by evanrichter** [comment]
It would let you label an output on the same line
`> ?en var_x:;p8 1 @ebp-0x11; ?en var_y:;p8 1 @ebp-0x17`

**Comment by radare** [comment]
then you dont need to flush anything, just another command after it that adds a newline at some point after that. pls fix the comments

**Comment by radare** [comment]
please use tabs to indent. your pr is full of unaligned spacy indentations

**Pull request merged**